dtfs-central-api/src/helpers/get-keying-sheet-calculation-tfm-facility-values.ts

```diff
@@ -46,5 +47,6 @@ export const getLatestTfmFacilityValues = async (
     dayCountBasis,
     interestPercentage,
     coverPercentage,
+    value,
```

Comment:
This PR has made this function used in the initial case so it is therefore fully in scope of this PR for this function to be consistent.

Reply:
I still do not think it is needed in this PR which is already quite large.  I understand the need in the future but right now, value is only used in one place which uses the snapshot value.  calculateInitialUtilisationAndFixedFee 
I don;t mind making another ticket and doing it there.  It will require 2 new helpers, 1 to remove replication and then added tests which is worthy of its own PR

Reply:
Resolving - I disagree but perhaps not important right now